pricegraph/src/lib.rs

```diff
@@ -224,6 +226,49 @@ mod tests {
         assert!(orderbook.bids.is_empty());
     }
 
+    #[test]
+    fn transitive_orderbook_simple() {
```

Comment:
Let's discuss based on this test, if this is the correct result (before I changed the other test too).

I think the price (0.5 and 2.0) is correct but is the multiplication/division with the fee factor correct or inversed?

Reply:
The fees should push the two prices further apart, so they *appear* to be correct... since `0.5 / FEE_FACTOR < 0.5` and `2.0 * FEE_FACTOR > 2.0`.

I think the issue with my reasoning on the prices is that we want the price to be expressed in `quote` token and in my head, that meant that it was `base_amount / quote_amount` which is inverted, and should be `quote_amount / base_amount`, so that if we multiply some quantity of `base`, we will get the total price in `quote`. Things appear to be correct now.